src/schema/nmdc.yaml

```diff
@@ -2968,6 +2968,12 @@ slots:
     abstract: true
     description: >-
       This is a grouping for any of the gold path fields
+    annotations:
+      tool_tip:
```

Comment:
I don't think I've come across this spelling of "tooltip" before. I don't meant to say it's incorrect, just that I think I've only ever seen it spelled as a single word.